pyproject.toml

```
@@ -65,7 +65,7 @@ repository = "https://github.com/codespell-project/codespell"
 
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools>=64", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=64", "setuptools_scm[toml]>=7.1.0"]
```

Perhaps I have misunderstood the issue at hand, but:
```suggestion
requires = ["setuptools>=64", "setuptools_scm[toml]<=7.1.0"]
```

---

good one, let me test that. 

The problem has been open since at least september 20th, see this [issue](https://github.com/codespell-project/codespell/issues/3088).

---

Tested and works fine. Good suggestion, this is more precise than what I proposed.